### PR TITLE
Added rastertstyler pulgin and example Close #547

### LINKED ISSUE #547: Create a raster style editor for layer
The style editor allows to create a color map for the raster layer, and have a preview on map.

### PR DESCRIPTION
**Pull request by kappu72**

### CODE REVIEW

File: web/client/examples/rasterstyler/assets/css/mobile.css [hunk 1]
```diff
@@ -0,0 +1,259 @@
+
+/* General Mobile Usability styles */
```
Comment (offtherailz) [inline web/client/examples/rasterstyler/assets/css/mobile.css]: This is not needed for the example

Comment (kappu72) [inline web/client/examples/rasterstyler/assets/css/mobile.css]: ok I'll remove

File: web/client/examples/rasterstyler/components/viewer/MadeWithLove.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/**
```
Comment (offtherailz) [inline web/client/examples/rasterstyler/components/viewer/MadeWithLove.jsx]: It is not needed in this sample

Comment (kappu72) [inline web/client/examples/rasterstyler/components/viewer/MadeWithLove.jsx]: fixed

**Pull request merged**